docs/source/ja/_toctree.yml

```
@@ -1,14 +1,146 @@
 - sections:
   - local: index
     title: 🤗 Transformers
+  - local: quicktour
+    title: クイックツアー
   - local: installation
     title: インストール
-  title: はじめに
+  title: Get started
 - sections:
+  - local: pipeline_tutorial
+    title: パイプラインを使用して推論を実行する
+  - local: autoclass_tutorial
+    title: AutoClass を使用して移植可能なコードを作成する
+  - local: preprocessing
+    title: データの前処理
+  - local: training
+    title: 事前トレーニングされたモデルを微調整する
+  - local: run_scripts
+    title: スクリプトを使用してトレーニングする
   - local: accelerate
-    title: 🤗 Accelerate を用いた分散学習
-  title: チュートリアル
+    title: 🤗 Accelerate を使用して分散トレーニングをセットアップする
+  - local: peft
+    title: 🤗 PEFT を使用してアダプターをロードしてトレーニングする
+  - local: model_sharing
+    title: モデルを共有する
+  - local: transformers_agents
+    title: エージェント
+  - local: llm_tutorial
+    title: LLM を使用した生成
+  title: Tutorials
+  sections:
```

This is breaking the `build pr doc` check:

```suggestion
 - sections:
```